js/index.js

```diff
@@ -1 +1,48 @@
 // Your code goes here
+// `mouseover`
+// `keydown`
+// `wheel`
+// `drag / drop`
+// `load`
+// `focus`
+// `resize`
+// `scroll`
+// `select`
+// `dblclick`
+
+// Selector Elements
+const header = document.querySelector('.main-navigation');
+const navLinks = document.querySelectorAll('.nav-link');
+const blog = document.querySelectorAll('.logo-heading');
+const imgOne = document.querySelector('.img-content img');
+const imgTwo = document.getElementsByTagName('img')[3];
+const body = document.querySelector('body');
+
+// const header = document.querySelector('.main-navigation');
+// const header = document.querySelector('.main-navigation');
+// const header = document.querySelector('.main-navigation');
+
+// Event Listener Methods
+// Mouseover
+header.addEventListener('mouseover', event => (header.style.backgroundColor = 'dodgerblue'));
+header.addEventListener('mouseout', event => (header.style.backgroundColor = 'white'));
+navLinks.forEach(el => {
+	el.addEventListener('mouseover', event => (event.target.style = 'color: white'));
+});
+navLinks.forEach(el => {
+	el.addEventListener('mouseout', event => (event.target.style = 'color: red'));
+});
+
+// keydown ... not responding
+//blog.addEventListener('keydown', alert("You're about to leave the site."), false);
```

Comment:
In order for the keydown event to work the element it is applied to be needs to be in focus. 

If you want to apply it generally to the page you would need to add an event listener to the document object.

```document.addEventListener('keydown', callbackFunction)```

Also currently you are calling a function as the second argument, this needs to be a callback function i.e.

```() => alert("You're about to leave the site.")```